packages/firestore/test/integration/api/database.test.ts

```diff
@@ -1600,4 +1601,31 @@ apiDescribe('Database', (persistence: boolean) => {
       });
     });
   });
+
+  it('can set and get data with auto detect long polling enabled', () => {
+    const settings = {
+      ...DEFAULT_SETTINGS,
+      experimentalAutoDetectLongPolling: true
+    };
+
+    return withTestDbsSettings(
+      persistence,
+      DEFAULT_PROJECT_ID,
+      settings,
+      1,
+      async ([db]) => {
+        const doc = await db.collection('users').doc();
+
+        const data = { name: 'Rafi', email: 'abc@xyz.com' };
```

Comment:
```suggestion
        const data = { name: 'Rafi', email: 'abc@xyz.com' };
        const doc = await db.collection('users').doc();
```

... but this is just a personal preference that you may ignore :)